rpcserver.go

```diff
@@ -456,7 +456,7 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 
 		// If the asset meta field was specified, then the data inside
 		// must be valid. Let's check that now.
-		if err := seedling.Meta.Validate(); err != nil {
+		if err = seedling.Meta.Validate(); err != nil {
```

Comment:
I don't really understand why we don't want shadowing here? We tend to do this everywhere and as far as I'm aware there shouldn't be any side effect.